### PR TITLE
Feature/compile with mini portile2

### PR DESCRIPTION
**Pull request by CAMOBAP**

### CODE REVIEW

File: rakelib/libxar.rake [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+task :libxar do
+  unless File.exist?(checkpoint)
+    recipe.cook
+    touch checkpoint
+  end
+
+  recipe.activate
+
+  FileUtils.cp(File.join(__dir__, "..", recipe.port_path, "lib", "libxar.dylib"), "lib/xar/")
+end
```
Comment (alemorozov) [inline rakelib/libxar.rake]: Looks awesome! 👍

File: xar.gemspec [hunk 2]
```diff
@@ -24,7 +24,9 @@ Gem::Specification.new do |spec|
...
   spec.add_runtime_dependency "ffi", "~> 1.14"
   spec.add_runtime_dependency "ffi-stat", "~> 0.3"
-  spec.add_runtime_dependency "rake-compiler", "~> 1.1"
-  spec.add_runtime_dependency "rake", "~> 10"
-  spec.add_development_dependency "bundler", "~> 2.2"
+
+  spec.add_development_dependency "rake", "~> 10"
+  spec.add_development_dependency "bundler"
+  spec.add_development_dependency "mini_portile2"
+  spec.add_development_dependency "rake-compiler-dock", "~> 1.1"
```
Comment (alemorozov) [inline xar.gemspec:31]: `rake-compiler-dock` helps with local testing?

File: rakelib/libxar.rake (hunk 1 above)
Comment (CAMOBAP) [reply rakelib/libxar.rake]: Almost finished cross compilation will update PR today

File: xar.gemspec (hunk 2 above)
Comment (CAMOBAP) [reply xar.gemspec:31]: nope, this for cross-compilation with docker

**Comment by alemorozov** [comment]
I think it can be merged now. Then I propose to rebase everything in the `cross-compilation` branch.

**Comment by CAMOBAP** [comment]
@alexeymorozov please give me 1-2h to finish it

**Pull request merged**